### PR TITLE
Add PR and issue templates

### PR DESCRIPTION
**Pull request by smoia**
This should appear as a co-authorship. If it doesn't, I'm going to check why.

### CODE REVIEW

**Comment by tsalo** [comment]
WDYT about adding a NeuroStars link for usage questions, like we did for `tedana`?

**Comment by smoia** [comment]
Can you explain a bit more - like, in what does it consists?
(Also, I'm not sure I want to spread the channels of communication more than we do now. That's a question for the community, but we can ask it at the dev call in February)

**Comment by tsalo** [comment]
The idea is just to point usage questions toward NeuroStars instead of issues. As the issue template selection page, there's a link to the NeuroStars `tedana`-tagged topics page, so that users can see what questions have already been asked and, if they do open a new topic, the `tedana` tag is automatically added. See below for how it looks:

Screen Shot 2021-01-25 at 11 14 43 AM

One benefit to having usage questions on NeuroStars is that ones which relate to multiple `physiopy` repositories will be easier to track since they won't be linked to a specific repo.

**Comment by smoia** [comment]
Fair enough! @physiopy/all what do you think?
Would there be someone monitoring the whole time Neurostars though? (I guess we can set up alerts, but the question is: once people ask, is there someone that will answer?)

**Comment by smoia** [comment]
@tsalo I *think* I added the link to neurostars, but if you could have a look at it, you'd do a big favor!

File: .github/ISSUE_TEMPLATE/ISSUE_TEMPLATE_FEATURE_REQUEST.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Use this template to request new features.
+title: ''
+labels: Enhacement
```
Comment (tsalo) [inline .github/ISSUE_TEMPLATE/ISSUE_TEMPLATE_FEATURE_REQUEST.md:5]:
```suggestion
labels: Enhancement
```

**Review by tsalo — CHANGES_REQUESTED** [review]
I just noticed one minor typo. The NeuroStars link looks good!

**Review by tsalo — APPROVED** [review]
LGTM!

**Pull request merged**